README.md

```diff
@@ -1,7 +1,29 @@
-[![Build Status](https://travis-ci.org/brndnmtthws/hdfs.svg?branch=master)](https://travis-ci.org/brndnmtthws/hdfs)
+[![Build Status](https://travis-ci.org/mesosphere/hdfs.svg?branch=master)](https://travis-ci.org/mesosphere/hdfs)
 HA HDFS on Apache Mesos
 ======================
-aka 'Super Available' HDFS
+Starts 1 active NameNode (with JournalNode and ZKFC), 1 standby NN (+JN,ZKFC), 1 JN, and everything else is DataNodes.
+
+Building HDFS-Mesos
 --------------------------
+1. `mvn package`
```

Comment:
It is a small code base so building is very quick. Running 'mvn clean
package' is a good practice to make sure all cached output is cleared and
only the most current code is in use. Since it won't take much time, there
is not much of a downside and this will allow us to be safe and avoid
strange bugs.

It seems that combining the building of the java project with the library
bundling in one script is best so the user can run one script to build the
latest code.


On Wed, Dec 31, 2014 at 1:27 AM, adam-mesos <notifications@github.com>
wrote:

> In README.md
> <https://github.com/mesosphere/hdfs/pull/34#discussion-diff-22377763>:
>
> >  --------------------------
> > +1. `mvn package`
>
>
>    1. Why run mvn clean for every build? The compiler should be able to
>    support incremental builds, so mvn clean is only to guarantee that you
>    are recompiling every single file from scratch. This slows down the compile
>    time, often unnecessarily. If the developer wants to start clean, they can
>    run mvn clean and/or build-hdfs clean before building as usual.
>    2. I initially wanted to split out the building of the java project (mvn
>    package) from the library bundling in build-hdfs (which I considered
>    renaming to bundle-hdfs), but I can understand running mvn package
>    inside build-hdfs, since you can always just run mvn package manually
>    for a pure project build, and then run build-hdfs afterwards to bundle
>    it up. (If we're not doing mvn clean inside build-hdfs, then
>    re-running mvn package is a noop.)
>    3. If somebody still wants to run the clean build as one command as
>    before, they can just do mvn clean && ./bin/build-hdfs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/hdfs/pull/34/files#r22377763>.
>